app/browser/menu.js

```diff
@@ -20,7 +20,7 @@ const settings = require('../../js/constants/settings')
 
 // State
 const {getByTabId} = require('../common/state/tabState')
-const tabState = require('../../app/common/state/tabState')
+const tabState = require('../common/state/tabState')
```

Comment:
weird; the path before was invalid, right? did references before to `tabState` work (before this PR)?

Reply:
it was not invalid, we just went one more step up and then back into the app folder

Reply:
where now we just go one folder up and then into common

Reply:
ah- I totally see it now 😄 Was up past bedtime and didn't catch that LOL

Reply:
yep, just some tiny clean up whilst there